src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int PARTICULAR_DAY = 0;
+    private static final int NAME_OF_EMPLOYEE = 1;
+    private static final int WORKING_HOUR = 2;
+    private static final int INCOME_PER_HOUR = 3;
+
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        LocalDate dtFrom = LocalDate.parse(dateFrom, formatter);
+        LocalDate dtTo = LocalDate.parse(dateTo, formatter);
+        StringBuilder builder = new StringBuilder("Report for period ");
```

Comment:
For the furute, do not use such naming, how to guess what it stores?
```suggestion
        StringBuilder report = new StringBuilder(HEADER);
```